pom.xml

```diff
@@ -228,7 +235,7 @@
         <dependency>
             <groupId>org.mule.sdk</groupId>
             <artifactId>mule-sdk-api</artifactId>
-            <version>0.6.0</version>
+            <version>0.7.0</version>
```

Comment:
Why is this being upgraded? We will need it in the future  but i dont understand why now

Reply:
Because taking into account the playbook https://docs.google.com/document/d/1PBMVwFhtQ8XCCi1myTWz09NbvjWcUc8VrySqdeCvZBk/edit#heading=h.qxy1o8fa0ebt it necessary for run over Java 17 

<img width="867" alt="Screenshot 2023-11-22 at 10 09 44" src="https://github.com/mulesoft/mule-file-connector/assets/157246/a14dc6a7-51b2-4afc-9711-d55c5dc0171e">

Reply:
It is needed to add the annotation to communicate java 17 support, that exceeds the scope of this  task i think

Reply:
Ok, Thanks, fixed!